sway/commands.c

```diff
@@ -1124,9 +1124,26 @@ static struct cmd_results *cmd_bar(int argc, char **argv) {
 	bar->tray_padding = config->bar.tray_padding;
 	list_add(config->bars, bar);
 
+	// set bar id
+	int i, n;
+	int len = 5; // "bar-" + \0
```

Comment:
Anyone want to take bets on when we'll get the first bug report for screens >1,000,000 pixels wide?